printer_zpl2/lib/zpl2.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 SYLEAM (<http://www.syleam.fr>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+class Zpl2(object):
```

Comment:
The class has been developped with the module since a long time, but as it is really generic, creating a separate python lib is a good idea.
I'll do that :)